src/components/Categories/Categories.component.tsx

```diff
@@ -1,34 +1,17 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 
 import CategoryCard from '../CategoryCard';
-import productCategories from '../../mocks/en-us/product-categories.json';
-import { Category } from '../../utils/types';
+import { useCategories } from '../../utils/hooks/useCategories';
 import { CategoriesWrapper, Title, CategoriesList } from './Categories.styled';
 
 function Categories() {
-  const [categories, setCategories] = useState<Category[]>([]);
-
-  useEffect(() => {
-    const data = productCategories.results;
-
-    const getCategories = data.map((item) => {
-      const id = item.id;
-      const title = item.data.name;
-      const img = item.data.main_image.url;
-      return {
-        id,
-        title,
-        img,
-      };
-    });
-    setCategories(getCategories);
-  }, []);
+  const { data } = useCategories();
```

Comment:
We should use more descriptive names, for example instead of "data" we could use "categories".